scss/_navbar.scss

```diff
@@ -77,9 +77,8 @@
   }
 
   .dropdown-menu {
-    position: static !important;
+    position: static;
```

Comment:
`.dropdown-menu-right` only exist here, because we handle that on our JS for the other Dropdown which aren't in a navbar.
But yes I think it's possible to add a media query just for the Dropdown position